src/main/java/org/jacp/repository/BaseRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.jacp.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.NoRepositoryBean;
+
+/**
+ * @author saffchen created on 31.07.2023
+ */
+@NoRepositoryBean
+public interface BaseRepository<T> extends JpaRepository<T, Long> {
```

Comment:
Why this interface is required?

Reply:
To possibly add functionality with the addition of other tables